package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sentry/fullstory",
-  "version": "1.0.2",
+  "version": "1.1.0",
```

Comment:
@patrick-fs Wondering if we should include a changelog file with this change now

Reply:
@scefali I've added a chagngelog